windows/setup/test_setup.yml

```diff
@@ -72,3 +72,11 @@
 - name: "Take a base snapshot if it does not exist"
   include_tasks: create_base_snapshot.yml
   when: not base_snapshot_exists
+
+- name: "Set fact of whether VM boot disk controller is known"
+  ansible.builtin.set_fact:
+    vm_boot_disk_ctl_got: "{{ win_boot_disk_ctl_type is defined and win_boot_disk_ctl_type }}"
```

Comment:
I suppose this fact can be defined at the end of `win_get_boot_disk_ctl_type.yml` like `guest_os_system_info_retrieved` defined in `get_windows_system_info.yml`.

Reply:
`win_boot_disk_ctl_type` may be set in `deploy_vm` test case directly.